flight/mcu/lib/FifoBuffer/FifoBuffer.h

```diff
@@ -1,67 +1,106 @@
-//
-// Created by 16182 on 9/27/2021.
-//
-
 #ifndef TARS_SOFTWARE_FIFO_H
 #define TARS_SOFTWARE_FIFO_H
 
 #include <cstdint>
 #include <cstring>
+#include <ChRt.h>
```

Comment:
you need to include iterator to use std::input_iterator_tag

Reply:
Fixed, although it seems to compile fine without it, which doesn't make sense